site/content/en/references/kustomize/kustomization/replacements/_index.md

```diff
@@ -198,10 +198,14 @@ Strings are used for mapping nodes. For sequence nodes, we support two options:
 
 2. Index by key-value pair: `spec.template.spec.containers.[name=nginx].image`
 
+And, Wildcard( `*` ) is replacing every element.
```

Comment:
```suggestion
3. Index with a wildcard match: `spec.template.spec.containers.*.env.[name=TARGET_ENV].value`. This will target every element in the list.
```